src/file/file.service.spec.ts

```diff
@@ -489,6 +489,294 @@ describe('File Logic', () => {
 
   });
 
+  describe('#createShortcut', () => {
+    // create file section
+
+    it('should throw an error when key is not sent with file', async () => {
+      await FileService.create(bucket, 'myFolder', USER.id, 'Text', 'drive', '', '', size)
+        .should.eventually.be.rejectedWith(ServerError, 'No key sent');
+    });
```

Comment:
It is incorrect. File key should not be sent with shortcut

Reply:
okay, what about the bucket?

Reply:
done